test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -145,6 +145,7 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
     attachment = result.dig(:editions, 0, :attachments, 0)
 
     assert_equal publication_file.attachments.first.url, attachment[:url]
+    assert_match(/^file\-attachment\-title\-[0-9]$/, attachment[:title])
```

Comment:
I'd like to double check that we don't expect each variant of FileAttachment to have its own title (e.g. "File attachment PDF" vs "File attachment thumbnail").

Currently I believe we just have `:title` on the FileAttachment root, which in this hypothetical scenario would be "File attachment".